### PR TITLE
LPS-164602 | Add Autom. test FDSCustomView#ManagementBarCanBeLocalized

### PR DESCRIPTION
**Pull request by eltonccdantas**
**Test Plan**

- Given: localized FDS sample portlet
- And Given: site URL localized to Spanish //localhost:8080/es/*
- When: on customized tab view
- Then: Filter button is localized //assert Filtro is present in first navbar-nav element
- And Then: Search help text is localized //assert Buscar is present in search field
- And Then: Change Display dropdown items are localized
- And Then: Custom Views dropdown items are localized
- And Then: Ellipsis dropdown "Save" is localized
- And Then: Reset Filters link is localized

**JIRA Ticket:
https://issues.liferay.com/browse/LPS-164602**

### CODE REVIEW

**Comment by eltonccdantas** [comment]
ci:test:sf

**Comment by eltonccdantas** [comment]
ci:test:sf

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase [hunk 1]
```diff
@@ -191,4 +189,56 @@ definition {
 		}
 	}
 
+	@description = "LPS-164602 Frontend Dataset - column labels are not localized "
+	@priority = "4"
+	test ManagementBarCanBeLocalized {
+		task ("site URL localized to Spanish //localhost:8080/es/ and on a customized tab view") {
```
Comment (john-co) [inline modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase:195]: Include BDD terms to all tasks ie: `And Given`

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FDSFilters.macro [hunk 2]
```diff
@@ -100,6 +100,19 @@ definition {
...
+				locator1 = "FrontendDataSet#RESET_FILTERS",
+				reset_filter = "Reset Filters");
+		}
+		else {
+			ClickNoError(
+				locator1 = "FrontendDataSet#RESET_FILTERS",
+				reset_filter = "${reset_filter}");
+		}
+	}
+
```
Comment (john-co) [inline modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FDSFilters.macro:115]: Not bad

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase [hunk 3]
```diff
@@ -34,8 +34,6 @@ definition {
 	}
 
 	tearDown {
-		var testPortalInstance = PropsUtil.get("test.portal.instance");
```
[the remark this answers is not in the document]
Comment (eltonccdantas) [reply modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase]: Sorry, my mistake, I've already updated.

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase (hunk 1 above)
Comment (eltonccdantas) [reply modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase]: Ok,

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FDSFilters.macro (hunk 2 above)
Comment (eltonccdantas) [reply modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FDSFilters.macro:115]: Thanks!

**Comment by eltonccdantas** [comment]
ci:test:frontend:nocompile

**Comment by eltonccdantas** [comment]
> Some minor edits, but besides that, looks good to me! sunglasses
> 
> Just make sure the tests pass. There seems to be some rebase issue. You can checkout your master branch, run `git reset --hard upstream/master` and rebase your LPS branch against your master. Let us know if this doesn't work. cc/ @mateusralv

It says it's already up to date.

**Comment by eltonccdantas** [comment]
ci:test:sf

**Comment by eltonccdantas** [comment]
ci:test:frontend:nocompile

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase [hunk 4]
```diff
@@ -191,4 +191,56 @@ definition {
...
 
+	@description = "LPS-164602 Frontend Dataset - column labels are not localized "
+	@priority = "4"
+	test ManagementBarCanBeLocalized {
+		task ("And Given: site URL localized to Spanish //localhost:8080/es/ and when on a customized tab view") {
+			var portalURL = PropsUtil.get("portal.url");
+
+			Navigator.openSpecificURL(url = "${portalURL}/es/frontend-data-set-test-page/");
+		}
+
```
Comment (mateusralv) [inline modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase:202]: Hey @eltonccdantas could you add `Navigator.gotoPage(pageName = "Frontend Data Set Test Page");` here? By [poshi report ](https://testray.liferay.com/reports/production/logs/2022-10/test-1-38/test-portal-acceptance-pullrequest(master)/4534/functional-tomcat90-mysql57-jdk8/0/16/LocalFile.FDSCustomView_ManagementBarCanBeLocalized/index.html.gz?authuser=0) When it changes the language of the page is being directed to  home page and this is breaking test

**Comment by eltonccdantas** [comment]
ci:test:frontend

**Comment by eltonccdantas** [comment]
ci:test:frontend:nocompile

**Comment by eltonccdantas** [comment]
ci:test:frontend:nocompile

**Comment by eltonccdantas** [comment]
ci:test:frontend:nocompile

**Comment by mateusralv** [comment]
The [failures](https://github.com/liferay-frontend/liferay-portal/pull/2812#issuecomment-1295649168) are not related, seems good to forward 🎉 🚀

**Comment by mateusralv** [comment]
ci:forward

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path [hunk 5]
```diff
@@ -160,7 +170,7 @@
 </tr>
 <tr>
 	<td>RESET_FILTERS</td>
-	<td>//*[contains(@class,'management-bar')]//*[@type='button' and contains(text(), 'Reset Filters')]</td>
+	<td>//*[contains(@class,'management-bar')]//*[@type='button' and contains(text(), '${reset_filter}')]</td>
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path:173]: for variables within *.path files, our naming convention is to start with `key_`

**Pull request closed without merging**